src/runtime-rs/crates/resource/src/volume/share_fs_volume.rs

```diff
@@ -231,6 +245,17 @@ impl Volume for ShareFsVolume {
     fn get_device_id(&self) -> Result<Option<String>> {
         Ok(None)
     }
+
+    fn get_guest_path(&self) -> Result<Option<String>> {
```

Comment:
I think you don't need this trait, if copy file is implemented in this file, the `guest path` will be stored in mount, `get_volume_mount` will be enough for use.